package.json

```diff
@@ -55,9 +55,9 @@
     "ts-node": "ts-node",
     "prettier": "prettier",
     "// - APPS": "Run Apps in dev and with packaged modules",
-    "start": "ng serve --aot --open",
+    "start": "npm run serve:hmr:integration",
```

Comment:
Probably use `yarn` here as our deps are locked via `yarn`?

Reply:
Changed it to yarn